config/routes.rb

```diff
@@ -36,6 +36,7 @@
   get '/privacy',   to: 'pages#privacy_policy', as: :privacy_policy
   get '/partners',  to: 'pages#partners',       as: :partners
   get '/hot',       to: 'pages#hotsauce',       as: :hotsauce
+  get '/loginpage', to: 'pages#login',          as: :login
```

Comment:
This is my bad for not writing this down anywhere, but we canonically use the term "sign in" over "log in". I'd also add that having "page" in the path is redundant, so I think `/signin` and `:signin` would be the best fit.